libsrc/c64/crt0.s

```diff
@@ -3,13 +3,13 @@
 ;
 
         .export         _exit
+        .exportzp       init_load_, init_run_
         .export         __STARTUP__ : absolute = 1      ; Mark as startup
+
         .import         initlib, donelib
-        .import         zerobss
-        .import         callmain
+        .import         move_init, zerobss, callmain
         .import         RESTOR, BSOUT, CLRCH
```

Comment:
As you seem to do a general overhauling of this file I suggest to remove the unrefenced `RESTOR` and `CLRCH`.